rfcs/0004-replace-unicode-quotes.md

```diff
@@ -0,0 +1,64 @@
+---
+feature: replace-unicode-quotes
+start-date: 2017-03-19
+author: layus
+co-authors: (find a buddy later to help our with the RFC)
+related-issues:
+    - https://github.com/NixOS/nix/pull/1140
+    - https://github.com/NixOS/nix/issues/915
+    - https://github.com/NixOS/nix/pull/910
+---
+
+# Summary
+[summary]: #summary
+
+Nix uses unicode glyphs to quote strings and paths in its output.
+This RFC proposes to use only ASCII `"` and `'` for quoting purposes in strings
+printed during evaluation.
+
+# Motivation
+[motivation]: #motivation
+
+There are three main reasons for this change.
+
+1. _Correctness_: By removing preventively these characters, we will not have
+   to track the triggered issues separately. Unicode interact badly with
+   variable interpolation in bash and will create more issues if we keep them.
+2. _Compatibility_: Most terminal emulators do not recognise unicode quotes as
+   string delimiters. This makes string copy/paste from the terminal clumsy.
```

Comment:
Are there specific issues to point to? Are there specific fonts that's don't have the character defined?